CVLByExample/ExtensionContracts/Base2.sol

```diff
@@ -0,0 +1,26 @@
+contract Base2 {
```

Comment:
why do we need another file? what does this one demonstrate that it is not in the base1?

Reply:
Basically we can show that we can apply a summarization for the same extension method only for a specific base and not for other base.